pkg/controller/pod_volume_backup_controller.go

```diff
@@ -228,10 +229,11 @@ func (c *podVolumeBackupController) processBackup(req *velerov1api.PodVolumeBack
 	)
 
 	// if there's a caCert on the ObjectStorage, write it to disk so that it can be passed to restic
-	caCert, err := restic.GetCACert(c.backupLocationLister, req.Namespace, req.Spec.BackupStorageLocation)
+	caCert, err := restic.GetCACert(c.kbClient, req.Namespace, req.Spec.BackupStorageLocation)
```

Comment:
Would it be better to get the BSL object once and pass it to the subsequent functions?
This would require creating new APIs for this.
WDYT?

Reply:
That was how the last version of this PR was. I'd rather not make any more changes to this. This is as close as the original as it can be.

Reply:
Not the last version actually, but a couple versions ago.

Reply:
Yeah I agree the API was changed to accept a BSL object and that was a breaking API change.
I was hoping for a new API, not replacing the one that exists, to be created that works on the BSL object directly.

I think this change is still breaking the API. In that, I mean, if someone using this API in their code were to upgrade their dependency to a version of velero with this change, then their code wouldn't compile. No?

Reply:
Yes, it'll be a breaking change. I talked to @nrb about this, this is not an area of the code anyone should be importing. We'are also going to start moving things to inside a (to be added) `internal` package. Are you ok with this breaking change?

Reply:
As long as none of our plugins are using these functions, we should be OK.
Ideally we should be deprecating APIs across releases.

Reply:
@ashish-amarnath Yeah, that would be ideal, though we've got a lot of examples of where we move our code around without deprecating.

I think I'm ok with this change, personally, even though it is breaking with strict semantic versioning. We have a lot of "accidentally" public code that we've been pretty rude about moving around in the past, and I think it would be good to document our true, intentional public API.

Carlisia made https://github.com/vmware-tanzu/velero/issues/2654 to do this, and we'll ask folks to comment there as we move things into `internal`. If moving something breaks someone, we can revisit.

This definitely isn't ideal, but we've also got to be able to move forward.